sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -732,13 +704,11 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
           dir => (dir ** filter).get
         )
 
-        packageJsonFile +:
-          generatedWebpackConfigFile +:
-          customWebpackConfigFile ++:
-          entries.map(_._2) ++:
+        customWebpackConfigFile ++:
+          webpackResources.value.get ++:
           // Entries only contain launchers - we need to monitor
-          // Scala.js entrypoints themselves, too.
-          entryPointTask(stageTask).value +:
+          // Scala.js output and entrypoints themselves, too.
+          stageTask.value.data +:
```

Comment:
This change seems to revert the changes done in a previous commit in the same PR. Perhaps squash them together so that they never happened?